### PR TITLE
Use classes feature

### PR DESCRIPTION
**Pull request by emmanuellekamwa**
In this project as required in the requirements, we have made use of classes and methods to create the add and remove books functionality

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -1,9 +1,20 @@
...
-  this.author = author;
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
+
+  add() {
+    books.push(this);
+  }
```
Comment (vichuge) [inline script.js:13]: - [x] When I add a book, the inputs still have the value, please, delete it once you add a book. 🐱

File: script.js [hunk 2]
```diff
@@ -1,9 +1,20 @@
...
+
+  remove() {
```
Comment (vichuge) [inline script.js:15]: - [x] When I tried to remove books, this happens, please fix it. 🌵 
image
- [OPTIONAL] Maybe you'll have a problem removing elements according to the title if some elements have the same, consider using the id for each element and remove this in place title. 🦊

**Review by vichuge — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi dear student, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 
You used proper GitHub Flow ✔️
Your linter checks are all passing ✔️
Your README and PR message are descriptive ✔️

## Aspects to improve ♻️

- [x] It looks like you're using the `package.json` file here, but this project is not required to use webpack, I kindly invite you to delete it. `package-lock.json` too. 🐶
image
- [x] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mimipeshy — APPROVED** [review]
## Hi @Team 👋🏾 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](<https://microverse.zendesk.com/hc/en-us/articles/1500004088561>)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](<https://airtable.com/shrQAqnBwek5a0O0s>)._

**Pull request merged**